ghcide/src/Development/IDE/Plugin/CodeAction/ExactPrint.hs

```diff
@@ -321,7 +322,7 @@ extendImportViaParent df parent child (L l it@ImportDecl{..})
       let childRdr = L srcChild $ mkRdrUnqual $ mkVarOcc child
           isParentOperator = hasParen parent
       when hasSibling $
-        addTrailingCommaT (head pre)
+        addTrailingCommaT (headNote "Development.IDE.Plugin.CodeAction.ExactPrint.extendImportViaParent" pre)
```

Comment:
Remove

Reply:
Suggest instead changing the `not null` guard on line 316 into a pattern guard that matches on `pre` and extracts the head there.